keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -80,16 +83,37 @@ Admission is only comparing the cpu requests against the allocatable cpus. Since
 We propose to add a new `CPUManager` policy option called `strict-cpu-reservation` to the `static` policy of `CPUManager`.
 With this policy option, we remove the reserved cores from the list of all available cores at the stage of calculation DefaultCPUSet. As a result, burstable and best-effort containers are launched with a cpuset in which the reserved cores are excluded. 
 
+### User Stories (Optional)
+
+#### Story 1
+System daemons and interrupt processing are there to serve all processes including guaranteed workloads; they needs be isolated from bursty workloads.
+
+#### Story 2
+Isolating system daemons and interrupt processing is especially critical in all-in-one deployments where workloads are placed on combined master+worker(+HCI storage) nodes.
```

Comment:
can we rephrase these stories from user perspective and with more specifics? for example, is story 1 about DPDK applications? story 2 seems about hyperconverged setups. Can we mention that and adding more context (to both)?

Reply:
comment taken, will rephrase the use cases